.github/workflows/e2e.yml

```diff
@@ -151,7 +151,7 @@ jobs:
         uses: actions/cache@v4
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}-e2e
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}-backend
```

Comment:
e2e has different pom.xml with the backend module, as you can see it uses `dolphinscheduler-e2e/pom.xml` instead of `/pom.xml`

Reply:
It doesn't matter. The maven cache path is `~/.m2/repository`. The old dependency cache is retained and new dependencies are added.

Reply:
I have half a memory we have some error with the same cache key https://github.com/apache/dolphinscheduler/pull/14808 but I am not sure, maybe we can keep this change to see whether regression in the further

Reply:
Thanks for clarification.